### ISSUE TITLE
[Feature] Add matomo analytics

### ISSUE THREAD

**Issue by Sigri44** [RealToken-Community/dashboard-v2#73]
Need to implement data analytics for dashboard.

Guide :
https://matomo.org/faq/new-to-piwik/how-do-i-start-tracking-data-with-matomo-on-websites-that-use-react/

URI = //data.realtoken.community
SITE_ID = 1

Thx

**Comment by NandyBa** [comment]
I'm ok with this feature but only if we add a consent button.

**Comment by Sigri44** [comment]
Yes, it's possible. Even if no data is collected!? -_- they don't log the ip or the country, but obviously just a salted hash and a timestamp, so no idea if the visitor is unique/recurrent, just to calculate the traffic.

**Comment by NandyBa** [comment]
A self-hosted counter is acceptable if it relies on non-reversible data. Privacy is crucial, and collecting sensitive data like IP addresses or location, even if anonymized, can still pose risks to user privacy.

**Comment by Sigri44** [comment]
https://matomo.org/privacy/

**Comment by BenoistP** [comment]
Soon™

**Comment by BenoistP** [comment]
done in #109  included in #126
